source/manual/buckups.html.md

```diff
@@ -0,0 +1,112 @@
+---
+owner_slack: "#govuk-developers"
+title: Use of backups
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+type: learn
+---
+
+This manual provides a shared definition of a backup and documents our current use of backups summarising strategies and tools we use.
+
+> A backup is a process of creating a copy of the data stored in a database and storing it in a safe location to allow data recovery in case of a disaster, system failure or human error.
+
+# AWS RDS Backups
+
+## Point-in-time RDS backups
+
+Point-in-time backups for Amazon Relational Database Service (RDS) allow us to restore your databases to a specific point in time in the past. This feature is useful when we want to restore your database to a state before an unintended change or data loss occurred.
+
+When point-in-time backups for an RDS instance are enabled, in our case in the [terraform configuration](https://github.com/alphagov/govuk-aws/blob/main/terraform/modules/aws/rds_instance/main.tf), Amazon RDS automatically takes regular (daily) backups of the database and stores them in Amazon S3. 
+Each backup contains a snapshot of the database at a specific point in time. We can restore the database to any of these points in time, within the retention period which is set for 7 days.
+
+The retention period is the amount of time that Amazon RDS stores our backups.
+When we restore a database to a point in time, Amazon RDS creates a new instance and restores the database to the specified point in time. The new instance will have a new endpoint and downstream apps will need to be updated to point to it.
+
+### Manual RDS snapshots
+
+It is also possible to take manual snapshots at any time to create a backup. Amazon RDS snapshots are point-in-time backups of a relational database that are stored in Amazon S3. They capture the entire database at a specific point in time, including the data, configuration and transaction logs.
```

Comment:
I don't think these are stored in S3 (based on the "Export to Amazon S3" action that's available when viewing a manual snapshot).

Reply:
Good point, it's recommended but doesn't happen automatically. Removed that now and added a link to AWS docs as you suggested in a comment below.